src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUnitUtilsTest.java

```diff
@@ -31,20 +32,39 @@
     }
 
     @Test(groups={"bucket"})
-    public void testIsCloudStorageURL(){
-        Assert.assertTrue(BucketUtils.isCloudStorageUrl("gs://abucket/bucket"));
-        Assert.assertFalse(BucketUtils.isCloudStorageUrl("hdfs://namenode/path/to/file"));
-        Assert.assertFalse(BucketUtils.isCloudStorageUrl("localFile"));
+    public void testIsGcsUrl(){
+        Assert.assertTrue(BucketUtils.isGcsUrl("gs://abucket/bucket"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("hdfs://namenode/path/to/file"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("localFile"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("https://www.somewhere.com"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("http://www.somewhere.com"));
+        Assert.assertFalse(BucketUtils.isGcsUrl("https://www.somewhere.com"));
+    }
```

Comment:
Where's the unit test for `isHttpUrl()`?

Reply:
oh, I misread what you were asking, added